content/docs/implementation-notes.md

```diff
@@ -394,36 +393,36 @@ As a result, each internal instance, composite or host, now points to its child
 }
 ```
 
-In the DOM you would only see the `<div>`. However the internal instance tree contains both composite and host internal instances.
+Dalam DOM anda hanya akan melihat `<div>`. Namun diagram *instance* internal berisi komposit dan *instance* internal *host*.
```

Comment:
```suggestion
Dalam DOM Anda hanya akan melihat `<div>`. Namun diagram *instance* internal berisi komposit dan *instance* internal *host*.
```

anda -> Anda.

Please take a look at other parts where "Anda" is improperly capitalised as well.

Reply:
ready, thanks